src/mimeparser.rs

```diff
@@ -1990,6 +1986,17 @@ fn is_attachment_disposition(mail: &mailparse::ParsedMail<'_>) -> bool {
             .any(|(key, _value)| key.starts_with("filename"))
 }
 
+fn is_valid_deltachat_vcard(mail: &mailparse::ParsedMail) -> bool {
+    let Ok(body) = &mail.get_body() else {
+        return false;
+    };
+    let contacts = deltachat_contact_tools::parse_vcard(body);
+    if let Some(c) = contacts.first() {
```

Comment:
```suggestion
    // For now, we only handle vCards containing one single contact:
    if let Some(c) = tools::single_value(contacts) {
```
in order not to have to worry about long vCards with many contacts.

Reply:
I agree, otherwise a question arises why we ignore the remaining contacts